plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -1235,10 +1221,18 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase {
         return true
       }
 
+      if (detector.pointersCount > 1) {
+        return false
+      }
```

Comment:
@Chaoba Any reason we need to do this check? Because it removes our capability to do two finger pan gesture.

Reply:
Created a draft PR to enable two-finger pan at https://github.com/mapbox/mapbox-maps-android/pull/1280